src/vs/platform/terminal/common/terminal.ts

```diff
@@ -68,7 +198,11 @@ export interface IPtyService {
 
 	getLatency(id: number): Promise<number>;
 
-	restartPtyHost?(): Promise<void>;
+	setTerminalLayoutInfo(args: ISetTerminalLayoutInfoArgs): void;
+
+	getTerminalLayoutInfo(args: IGetTerminalLayoutInfoArgs): Promise<ITerminalsLayoutInfo | undefined>;
+
+	orphanQuestionReply(args: IOrphanQuestionReplyArgs): void;
```

Comment:
Let's figure out orphan stuff before we merge, removing if we don't need

Reply:
removed bc we don't need